en/core-api/e-wallet.md

```diff
@@ -302,49 +313,69 @@ You will get the **API response** like the following.
 
-### Show QR Code Image (on computer or tablet)
+ The user flow is different for web browser (on a computer or a tablet) and smartphone.
 
-To display transaction QR Code image, use the url from `generate-qr-code` actions retrieved from API response. Simplest way is to **"hotlink"** the image url, if the frontend is HTML, put the url in image tag `<img src="[QR CODE URL]">`, or display it on a similar component without downloading.
-If the frontend does not support such scenario, download the QR code image from that url, then display it on frontend.
+**Displaying *QR Code Image* (on computer or tablet)**
 
-Instruction Example for **QR Code** :
+The steps to be followed to display the QR code image:
 
-1. Tap **Pay using GoPay**
-2. QR code will appear on the next page
-3. Open any **QRIS compatible app** (e.g: Gojek) on your mobile phone
-4. Tap **Pay** then scan the QR Code
-5. Check and verify your payment details then tap **PAY**
-6. Complete your security PIN / verification
-7. Your transaction is finished
+1. URL`generate-qr-code`should be used. `generate-qr-code` can be derived from 'actions' (API Response).
+2. To hotlink the image URL is the simplest way to get the QR code image. In HTML frontend, you can put the URL in image tag (img src="[QR CODE URL]">) and directly get the image.
+3. If the frontend is other than HTML, the QR code image should be downloaded from the URL, and should be displayed on the frontend.
+
+**Instruction Example for *QR Code* :** 
+
+1. Select **Payment Mode**, for example GoPay.
+2. Click **Pay Now**. QR code will be displayed.
+3. Open any **QRIS compatible app**, installed in your phone, for example Gojek.
+4. Scan the **QR code**.
+5. Click **Pay**, after checking and verifying your payment details.
+6. Verify your **security pin** and finish your transaction.
 
 ![GoPay QR Instruction](./../../asset/image/core-api_gopay-qr-pay.png)
 
-### Create Redirect Link to Gojek Apps (on SmartPhone)
+**Creating Redirect Link to Gojek Apps (on smartphone)**
 
-To redirect customer to Gojek app, use the url from `deeplink-redirect` actions retrieved from API response. Then customer can be redirected via server-side redirect, using javascript like `window.location=[DEEPLINK URL]`, or using HTML link `<a href="[DEEPLINK URL]">Pay with GoPay</a>`.
+The customers can be redirected to Gojek app through URL. This URL can be obtained from `deeplink-redirect`, retrieved from `actions` (API Response). JavaScript such as `window.location=[DEEPLINK URL]`, or HTML link such as `<a href="[DEEPLINK URL]">Pay with GoPay</a>` can be used to redirect the customers via server-side redirect.
 
-Instruction Example for **Deeplink** :
+**The steps for *Deeplink* are as given below:**
 
-1. Tap **Pay using GoPay**
-2. You will be redirected to **Gojek** app
-3. Check and verify your payment details then tap **Pay**
-4. Enter your security **PIN**
-5. Your transaction is finished
+1. Select Payment mode, for example GoPay.
```

Comment:
Why remove the `**Pay using GoPay**` ? this is important part, because the previous paragraph mention it